### PR TITLE
Update public instance in developer documentation to Linode 1.1.2

### PR DESCRIPTION
**Pull request by rcknplyr**
Updated public instance in developer documentation from 1.1.0 Heroku instance to 1.1.2 Linode instance.

### CODE REVIEW

**Comment by themightychris** [comment]
The Linode instance is the main link at the top of this article, the Heroku link is provided as an alternative in case the main link were to be down.

Do we know whether that Heroku instance has been getting kept up-to-date by anyone? We've automated through github deployment of the `develop` branch to another heroku account, we could replicate that to automate updating that production alternative heroku instance on release alongside updating Linode

**Comment by jlubken** [comment]
Heroku deploy was back at v1.1.0, I'm updating to deploy develop right now and will likely go back to v1.1.1 once I confirm that the develop works.

File: docs/getting-started/try-online.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ This instance is subject to unpredictable load though, and work to stabilize its
 
 ## Alternatives
 
-### Public Heroku Instance
+### Public Linode Instance
 
-Try this link: https://pennchime.herokuapp.com/
+Try this link: https://penn-chime.phl.io/
```
Comment (jlubken) [inline docs/getting-started/try-online.md:11]: I think this is supposed to be an alternative to Linode on line 3. Ask @mdbecker if he would update the heroku instance to v1.1.2.

**Review by jlubken — CHANGES_REQUESTED** [review]
I think we want to keep the heroku reference.

**Comment by rcknplyr** [comment]
Understood. Will close this.

**Pull request closed without merging**